.github/workflows/publish.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install Node.js, NPM and Yarn
         uses: actions/setup-node@v3
         with:
-          node-version: '18.x'
+          node-version: '18.17.0'
```

Comment:
#### Reason to Change
`node-gyp` was failing due to version difference when building using electron